### ISSUE TITLE
Zoom for IT can't find group template

### ISSUE THREAD

**Issue by n0brainer** [autopkg/MLBZ521-recipes#1]
I was trying to use the Zoom for IT.jss recipe, but i keep getting errors that it can't find it.

**Comment by MLBZ521** [comment]
I'm assuming you attempting to use the Zoom For IT JSS recipe?

For the Group Template, did you set the override value for `GROUP_TEMPLATE` to a custom value?  (The "generic" value (`GroupTemplate.xml`) doesn't exist in my repo.)

**Comment by n0brainer** [comment]
i created an override but didn't change anything.

**Comment by MLBZ521** [comment]
You'll need to change the value for `GROUP_TEMPLATE` which should point to a valid <Group Template>.xml file somewhere in your recipe path.  This is the standard process when using autopkg recipes.

**Comment by grahampugh** [comment]
@MLBX521 I came across this issue because it was mentioned for another recipe in the Slack `#jss-importer` channel.

My personal view is that `.jss` recipes should conform to the standards set out in https://github.com/autopkg/jss-recipes#requirements-and-configuration (indeed, the reason I have so far held back my jss recipes from the public is because they do not conform to the standard model), though since you are not submitting these recipes to the `jss-recipes` repo, you are not obliged to do so.

This means that your recipes would have the standard keys that people are familiar with using - that is:

```
<key>GROUP_NAME</key>
<string>%NAME%-update-smart</string>
<key>GROUP_TEMPLATE</key>
<string>SmartGroupTemplate.xml</string>
```

When the values are thus set, people don't have to override the `GROUP_TEMPLATE` key manually to point to their existing `SmartGroupTemplate.xml`.

If changing these defaults is too onerous, I would ask you at least to consider clarifying in your repo's `README` that your recipes do not conform to the best practices and overrides will therefore need to be customised to fit back into the standard model.

**Comment by MLBZ521** [comment]
Hey Graham @grahampugh,

Honestly, I'm pretty sure I copied those "default values" that I'm using from someone else's recipes...

Honestly, I've thought about submitting my `.jss` recipes to the `jss-recipes` repo and I went to review the style guide recently...and at that time, I just didn't time to spend on it.  It is something on my future to do list that I would like to do though.

That said,  I don't mind making those changes to the recipes I have.  I honestly didn't think the `value`'s would matter to much as I do not use the default values at all; I was only concerned with the `key` names matching standards.

I won't make any promises on when I can get to it, I have several projects I'm focused on right now for work, but I'll do it asap.
